animate.md

```diff
@@ -0,0 +1,56 @@
+## GIPHY Animated Text Creation 
+<img src="images/new-feature.gif">
+
+### Dynamic Text Feature 
+
+This new service _creates_ animated text results for search queries where there are no matching results in GIPHY's library. These results are returned in a number of different animation styles giving your users a variety of options to best express themselves.  
+ 
+Here are some examples below: 
```

Comment:
Should we remove this line @cgmaier 
Since the examples show up later in the documentation?